apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Tests/Google.Cloud.Spanner.Data.Tests.csproj

```diff
@@ -32,7 +32,7 @@
   <ItemGroup>
     <ProjectReference Include="..\Google.Cloud.Spanner.Data.CommonTesting\Google.Cloud.Spanner.Data.CommonTesting.csproj" />
     <Compile Update="V1\SessionPoolTests.*.cs">
```

Comment:
Just to double check that the remaining V1 here is needed.

Reply:
Yup. But the other one was broken. Presumably the "depends on" name is relative to the source file.